module-4/backend/week_two.md

```diff
@@ -1,22 +1,63 @@
 ## Week Two - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+1. What's one difference between ES5 and ES6?
+  In ES5, an anonymous function is defined with the syntax:
+  `function(arguments) { }`
 
-Note: When you're done, submit a PR. 
+  Whereas in ES6, an anonymous function is defined using arrow syntax:
```

Comment:
what's the difference between the es5 anonymous function syntax and the es6 syntax? besides how it looks!